src/types/item.resolvers.js

```diff
@@ -0,0 +1,13 @@
+const item = (_, args, ctx) => {
+  return ''; // TODO: return the item from dummy data
+}
+
+
+export default {
```

Comment:
Mixing ES6 import/export and commonjs here a bit. I think we should probably just use ES6 modules. I'll make a separate PR for that :+1:

Reply:
Yeah I was wondering why the Apollo docs don't use es6 import/export :(